.env.example

```diff
@@ -20,3 +20,9 @@ LADI_THRESHOLD=1.5 # float tunes the anomaly threshold based on the max value of
 
 LOCAL_DATA_FILE=validation_data/Hadoop_2k.json
 LOCAL_RESULTS_FILE=/tmp/local-results.txt
+S3_KEY=<REPLACE_WITH_S3_KEY> # The s3 key
+S3_SECRET=<REPLACE_WITH_S3_SECRET> # The s3 secret
+S3_HOST=http://localhost:8080 # The s3 endpoint url
```

Comment:
Can we also add the `LAD` prefix to `S3` variables? 

Also, why is `S3_HOST` example different from others?

Reply:
@vpavlin Okay Fixed it. I'm marking this as resolved. Awaiting the merge if this is good.